master/buildbot/test/integration/test_www.py

```diff
@@ -139,10 +142,10 @@ def apiGet(self, url, expect200=True):
         if expect200 and pg.code != 200:
             self.fail("did not get 200 response for '%s'" % (url,))
 
-        defer.returnValue(json.loads(body))
+        defer.returnValue(json.loads(bytes2NativeString(body)))
 
     def link(self, suffix):
-        return self.url + 'api/v2/' + suffix
+        return unicode2bytes(self.url) + b'api/v2/' + suffix
```

Comment:
in setup(), self.url is already ensured to be bytes, so that unicode2bytes is not needed